docs/content/development/extensions-core/sample-client-sslcontext.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: doc_page
+---
+
+## Sample SSLContext Provider Module
+
+This module contains a sample implementation of [SSLContext](http://docs.oracle.com/javase/8/docs/api/javax/net/ssl/SSLContext.html)
+that will be injected to be used with HttpClient that Druid nodes use internally to communicate with each other. To learn more about
+Java's SSL support, please refer to [this](http://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/JSSERefGuide.html) guide.
+
+# Configuration
+
+|Property|Description|Default|Required|
+|--------|-----------|-------|--------|
+|`druid.client.https.algorithm`|SSL Algorithm/Protocol to use.|`null`|yes|
+|`druid.client.https.trustStoreType`|The type of the key store where trusted root certificates are stored.|`java.security.KeyStore.getDefaultType()`|yes|
```

Comment:
yes this is required. Why do you think its not required ? 
For required fields, do you want me to not specify any defaults. I did it because `java.security.KeyStore.getDefaultType()` is something fetched on runtime so if users want to use this then not sure how to do that through a config.